hack/istio/download-istio.sh

```diff
@@ -65,9 +65,16 @@ echo "Output Directory: ${OUTPUT_DIR}"
 if [ -z "${ISTIO_VERSION}" ]; then
   if [ -z "${DEV_ISTIO_VERSION}" ]; then
     # get the latest released version
-    VERSION_WE_WANT=$(curl -L -s https://api.github.com/repos/istio/istio/releases | \
-          grep tag_name | sed -e 's/.*://' -e 's/ *"//' -e 's/",//' | \
-          grep -v -E "(snapshot|alpha|beta|rc)\.[0-9]$" | sort -t"." -k 1.2g,1 -k 2g,2 -k 3g | tail -n 1)
+    LATEST_RESPONSE=$(curl -L -s https://api.github.com/repos/istio/istio/releases/latest)
```

Comment:
Not sure why this file is updated in this PR.  Did you mean for it to be updated here?  Also, are you sure "latest" is what we want?  I thought we wanted "highest".  Can't "latest" be some z-stream release for an older version?

Reply:
> Not sure why this file is updated in this PR. Did you mean for it to be updated here?

There were a number of opaque CI failures that originated from this file and talking to the github API. The current implementation hides any error that occurs and just results in `VERSION_WE_WANT` being empty which causes errors later in the script.

> Also, are you sure "latest" is what we want? I thought we wanted "highest". Can't "latest" be some z-stream release for an older version?

Yes you're right we don't want "latest" but rather "highest". It looks like the intermittent github api issues have passed and the frontend tests are running again. I'll revert this change and open up a separate PR for it.